meshtastic/interfaces/ble/connection.py

```
@@ -322,6 +340,26 @@ def __init__(
         self.state_lock = state_lock
         self.thread_coordinator = thread_coordinator
 
+    @staticmethod
+    def _get_connect_timeout(pair_on_connect: bool) -> float:
```

Ruff (0.15.4)

[error] Boolean-typed positional argument in function definition

(FBT001)